UnionFind/NumberOfProvinces.py

```
@@ -0,0 +1,30 @@
+                  if self.array[i]==left:
+                     self.array[i]=right 
+      def findCircleNum(self, isConnected: list[list[int]]) -> int:
+          n=len(isConnected)
+          self.array=list(range(n))
+          m=len(isConnected[0])
+          for i in range(n):
+              for j in range(m):
+                  if isConnected[i][j]==1 and i!=j:
+                      self.union(i,j)
+          return len(set(self.array))
+class TestApp:
```

Flake8 (7.2.0)

[error] expected 2 blank lines, found 0

(E302)